### ISSUE TITLE
`Invalid character` error persists on 5.10.1 in Prisma Studio

### ISSUE THREAD

**Issue by controversial** [prisma/prisma#23225]
### Bug description

Yesterday, on 5.10.0, I encountered the “Error: invalid character” bug from #23201 due to curly quotes in comments in my Prisma schema. After updating to 5.10.1, removing the old generated client, and running `prisma generate`, this error has disappeared from `prisma generate` and from normal query usage.

However, when launching `prisma studio` and running a query, I still encounter an “invalid character” error that it seems the 5.10.1 patch release missed:

<img width="1345" alt="Screenshot 2024-02-21 at 9 36 18 AM" src="https://github.com/prisma/prisma/assets/10377391/8dc556b3-b360-4b94-a58a-f53a03145d11">

### How to reproduce

1. Use a schema with non-ascii characters, such as [the one in `tests/functional/issues/23201-non-ascii-comments`](https://github.com/prisma/prisma/blob/210217909e221349153868c2f934c5113cff8727/packages/client/tests/functional/issues/23201-non-ascii-comments/prisma/_schema.ts)
2. Connect a database
3. Run `prisma studio`
4. Click on the name of a table to cause Prisma Studio to perform a query

### Expected behavior

Prisma Studio works with non-ascii schemas, as it does on prisma < 5.10.0

### Prisma information

[Example schema](https://github.com/prisma/prisma/blob/210217909e221349153868c2f934c5113cff8727/packages/client/tests/functional/issues/23201-non-ascii-comments/prisma/_schema.ts)

### Environment & setup

- OS: macOS
- Database: PostgreSQL
- Node.js version: v20.10.0

### Prisma Version

```
prisma                  : 5.10.1
@prisma/client          : 5.10.1
Computed binaryTarget   : darwin-arm64
Operating System        : darwin
Architecture            : arm64
Node.js                 : v20.10.0
Query Engine (Node-API) : libquery-engine 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 5a9203d0590c951969e85a7d07215503f4672eb9 (at node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.10.0-34.5a9203d0590c951969e85a7d07215503f4672eb9
Default Engines Hash    : 5a9203d0590c951969e85a7d07215503f4672eb9
Studio                  : 0.498.0
```

**Comment by janpio** [comment]
Thanks for the report, we can confirm and are working on a fix.

**Comment by millsp** [comment]
Hey @controversial, the fix is available in `5.10.2` which we just released. Thanks for the report!

**Comment by controversial** [comment]
Confirming it’s fixed on my end - thanks!
